pages/settings/activity.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <v-container>
+    <v-row>
+      <v-col cols="12" :offset-md="1" md="5" class="pt-0 pb-4">
+        <!-- Administrator settings -->
+        <v-list
+          v-if="$auth.user.Policy.IsAdministrator"
+          two-line
+          class="mb-4"
+          disabled
+        >
+          <v-list-item-group>
+            <v-list-item v-for="activity in activityList" :key="activity.Id">
+              <v-list-item-avatar>
+                <v-icon>mdi-shield-account-outline</v-icon>
```

Comment:
This should probably vary depending on `activity.Type` and `activity.Severity` (Change the icon depending on the type of activity and have it in a `v-avatar` or something, that changes color based on severity)

Reply:
what are probable values for severity and type? I can see severity has 'Trace' and 'Error'. And for type there are many.
How to manage all those ?
should conditions be managed there in html part only or add complex ones in vue code?

Reply:
For the Type, currently it's just a string and according to server folks, it's non trivial to change, so we can probably forget that for now :)

For the level, you have a type available as part of the API (`LogLevel`), which is the following:
```typescript
export enum LogLevel {
    Trace = 'Trace',
    Debug = 'Debug',
    Information = 'Information',
    Warning = 'Warning',
    Error = 'Error',
    Critical = 'Critical',
    None = 'None'
}
```

You can use a method in your vue component that returns the color from a given severity level. You'd call it like:
```html
<v-avatar
  :color="getColorFromSeverity(activity.Severity)"
>
      <!-- your icon here -->
</v-avatar>
```

Then have a method in the component named like this:
```typescript
getColorFromSeverity(severity: LogLevel): string
```
With a basic switch..case inside of it :)